### ISSUE TITLE
Truncate plugin names - starter showcase

### ISSUE THREAD

**Issue by shannonbux** [gatsbyjs/gatsby#6720]
@sw-yx, @fk and I need to decide what length to truncate the words before you can do this, unless you have any ideas!

**Comment by sw-yx** [comment]
ok well.. let me know what you think!

**Comment by fk** [comment]
Hey Shannon, this is concerning the plugin names in the list of filters in the sidebar, right?

**Comment by shannonbux** [comment]
Yes and in the detail page, too! Some of them are on two lines and look a
little funky. We could also come up with a different method to make them
look nice on the detail page that doesn't involve truncation.

**Comment by fk** [comment]
> We could also come up with a different method to make them look nice on the detail page that doesn't involve truncation.

👍 Let's do that! I think the same could work for the sidebar filter list (although this is a little trickier). I'll mock s/th up!

**Comment by fk** [comment]
Hey @shannonbux, I finally got around to update https://www.figma.com/file/G3x7aYuyigYaibEpyxMsKLcP/gatsbyjs.org%3Ashowcase-starters?node-id=0%3A61 to show how untrucated plugins would look in the sidebar filter list:

* I think the best we can do is indent the second (third, fourth…) line by the width of `*-` in case the `gatsby` prefix in the dependency name has been replaced (this _should_ be possible with some minor CSS hackery)
* I'm unsure if moving the count to the last line of the item really helps, but it's the "correct" way to typeset this

![image](https://user-images.githubusercontent.com/21834/44093938-1b1bdfda-9fd5-11e8-9f4e-291cc5118450.png)

This would cover touch devices where we don't have the ability to show more information (e.g. in a [tooltip](http://getbootstrap.com/docs/4.1/components/tooltips/#examples)) when hovering an element. For desktop, a tooltip could be a solution that goes well along with truncation, although it implies that the user has to hover a sidebar filter or detail view link before he can decide wether he wants to set the filter/visit the link.
